src/Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileImage/Components/LocalImagePreview.tsx

```diff
@@ -0,0 +1,59 @@
+import { Avatar } from "@artsy/palette"
+import { Media } from "Utils/Responsive"
+import { Either } from "Utils/typeSupport"
+
+interface LocalImagePreviewLoadedProps {
+  imageUrl: string
+  onLoad: (image: React.SyntheticEvent<HTMLImageElement, Event>) => void
+}
+
+type LocalImagePreviewProps = Either<
+  LocalImagePreviewLoadedProps,
+  { isLoading: boolean }
+>
+
+export const LocalImagePreview: React.FC<LocalImagePreviewProps> = ({
+  imageUrl,
+  onLoad,
+  isLoading,
+}) => {
+  if (isLoading) {
+    return (
+      <>
+        <Media lessThan="sm">
```

Comment:
Looks like we should support responsive variants for this component. Will ticket.

Reply:
Yes please! and can we pair on it? would love to work on it!

Reply:
Of course!